tests/test_core.py

```diff
@@ -165,10 +165,9 @@ def test_marc21_field_247_matching():
         '247__': [{'a': 'B'}],
     })
 
-    assert data['other_standard_identifier'][0]['standard_number_or_code'] \
-        == 'A'
-    assert data['former_title'][0]['title'] == 'B'
-    assert len(data) == 2
+    assert 'A' == data['other_standard_identifier']['standard_number_or_code']
```

Comment:
@jirikuncar I don't see how we can deal with that without adding some schema information to the GroupableOrderedDict.

Or it has to store different things in different cases depending on the input data.